.circleci/config.yml

```diff
@@ -136,7 +136,7 @@ jobs:
           at: "."
       - run:
           name: Run htmlproofer (external links only)
-          command: bundle exec scripts/htmlproofer --external --retry-external 3 --retry-external-delay 8 --url-ignore congress.gov --url-ignore lastpass.com 
+          command: bundle exec scripts/htmlproofer --external --retry-external 3 --retry-external-delay 8 --url-ignore '/.*gov$ / gm' '/^.*\.(pdf|PDF)$ / gm' 
```

Comment:
The way the script is set up currently, it's expecting these ignore patterns to be plain strings, not regex patterns.

https://github.com/18F/identity-site/blob/6d45d11724d0b9692fc2aec15d53b3efc649f357/scripts/htmlproofer#L15

We could switch over to constructing it from the parameter as an assumed pattern:

```rb
  opt.on('--url-ignore PATTERN') { |pattern| proofer_options[:url_ignore] << Regexp.new(pattern) }
```

It would require us to update this line to exclude the forward slashes:

```suggestion
          command: bundle exec scripts/htmlproofer --external --retry-external 3 --retry-external-delay 8 --url-ignore '\.gov$' --url-ignore '\.pdf$' 
```

Reply:
I don't think we want to be using `Regexp.quote`, since that will look for the literal `\.gov$`:

```
$ irb
> Regexp.quote('\.gov$')
=> "\\\\\\.gov\\$"
```

We should use `Regexp.new` instead.